docs/_src/usage/usage/languages.md

```diff
@@ -0,0 +1,178 @@
+<!---
+title: "Languages Other Than English"
+metaTitle: "Languages Other Than English"
+metaDescription: ""
+slug: "/docs/languages"
+date: "2020-11-05"
+id: "languagesmd"
+--->
+
+# Languages Other Than English
+
+Haystack is well suited to open-domain QA on languages other than English.
+While our defaults are tuned for English,
+you will find some tips and tricks here for using Haystack in your language. 
+
+##Retrievers
+
+The sparse retriever methods themselves(BM25, TF-IDF) are language agnostic.
+Their only requirement is that the text be split into words.
+The ElasticsearchDocumentStore relies on an analyzer to impose word boundaries,
+but also to handle punctuation, casing and stop words.
+
+The default analyzer is an English analyzer. 
+While it can still work decently for a large range of langauges,
+you will want to set it to your language's analyzer for optimal performace.
+In some cases, such as with Thai, the default analyzer is completely incompatible.
+See [this page](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-lang-analyzer.html) 
+for the full list of language specific analyzers.
+
+```python
+document_store = ElasticsearchDocumentStore(analyzer="thai")
+```
+
+The models used in dense retrievers are language specific. 
+Be sure to check language of the model used in your EmbeddingRetriever.
+The default model that is loaded in the DensePassageRetriever is for English.
+We are currently working on training a German DensePassageRetriever model.
```

Comment:
"We are currently working on training a German DensePassageRetriever model and know other teams who work on further languages."